src/lib/Preview.js

```diff
@@ -1402,8 +1443,17 @@ class Preview extends EventEmitter {
                 id: fileId,
             },
         };
+
         const headers = getHeaders({}, token, sharedLink, sharedLinkPassword);
 
+        if (this.options.enableAdvancedContentInsights) {
+            this.pageTracker.init(this.previewOptions.contentInsightsOptions, { apiHost, headers }, this.file);
+            data.additional_information = {
+                view_session: {
```

Comment:
Does this require a schema change in the analytics table?

Reply:
Yes, because we need to correlate data from ACI to Access Stats. The monolith changes and schema will be updated as well